scripts/create-repos/index.js

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+const path = require(`path`)
+const yargs = require(`yargs`)
+const fs = require(`fs-extra`)
+const fetch = require(`isomorphic-fetch`)
+
+const BASE_DIRECTORY = `examples`
```

Comment:
Longer term - I think this will be consolidated into just `starters`. I don't see the point of differentiating the two.